src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day_sun calendar--month-length_29">
```

Comment:
The BEM modifier syntax should be `block__element--modifier` instead of `block__element_modifier`. Update the class names 'calendar--start-day_sun' and 'calendar--month-length_29' to 'calendar--start-day--sun' and 'calendar--month-length--29' respectively.